cves/kernel/CVE-2020-25669.yml

```diff
@@ -227,7 +242,7 @@ subsystem:
     e.g.
         name: ["subsystemA", "subsystemB"] # ok
         name: subsystemA # also ok
-  name:
+  name: drivers
   note:
```

Comment:
Saying "drivers" alone doesn't quite tell us where the mistake was. Could you possibly go a bit more in detail on that? For example "this was located in the drivers directory found in files ....."